src/suggestions.js

```diff
@@ -157,6 +160,6 @@ export default class SimpleteSuggestions extends HTMLElement {
 	}
 
 	get root() {
-		return this.closest("simplete-form");
+		return this.closest(`${this.rootSelector}`);
```

Comment:
I believe the entire logic should be encapsulated within this getter; no need to keep parts in the constructor?
```suggestion
		let selector = this.getAttribute("root-selector") || DEFAULTS.rootSelector;
		return this.closest(selector);
```

Note that I'm ditched the `data-` prefix: It's a custom element, so we're free to use custom attribute names.

Reply:
I was just trying to add configuration in a similar way to the other properties for the component (which use `data-item-selector`, `data-field-selector`, and `data-result-selector`). 

But I have no qualms about using a custom attribute name.

Reply:
Implemented this suggestion in the latest commit

Reply:
If I recall correctly (it's been a while... ), those `data-` attributes are necessary because the server typically provides regular (standardized) elements. Nevertheless, thanks for taking consistency into account there!